Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -410,5 +494,23 @@ private static bool AcceptedTokenVersionMismatch(MsalUiRequiredException msalSev
             // This is subject to change in the future
             return (msalSeviceException.Message.Contains("AADSTS50013"));
         }
+
+        /// <summary>
+        /// Gets an IAccount for the current B2C policy in the user claims
+        /// </summary>
+        /// <param name="accounts"></param>
+        /// <param name="policy"></param>
+        /// <returns></returns>
+        private IAccount GetAccountByPolicy(IEnumerable<IAccount> accounts, string policy)
```

Comment:
Let's do `GetAccountByUserFlow(IEnumerable<IAccount> accounts, string userFlow)` and update the xml